IntegrationTests/IntegrationTests.xcodeproj/xcshareddata/xcschemes/IntegrationScenarios.xcscheme

```diff
@@ -56,8 +56,8 @@
    </BuildAction>
    <TestAction
       buildConfiguration = "Release"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
-      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      selectedDebuggerIdentifier = ""
+      selectedLauncherIdentifier = "Xcode.IDEFoundation.Launcher.PosixSpawn"
```

Comment:
question/ Not sure on these changes - they do not look necessary.

Reply:
Hrm, yeah, probably just some changes from when I was testing crashes. I'll revert them.